core/rfb.js

```diff
@@ -329,16 +330,33 @@
 
         // Send a key press. If 'down' is not specified then send a down key
         // followed by an up key.
-        sendKey: function (keysym, down) {
+        sendKey: function (keysym, code, down) {
```

Comment:
this is breaking public API, so it'll need a release note (once we hit 1.0, which I'm hoping to soon, we can't break API this easily).